addins/Cake.CakeMail.yml

```diff
@@ -2,7 +2,7 @@ Name: Cake.CakeMail
 NuGet: Cake.CakeMail
 Assemblies:
 - "/**/Cake.CakeMail.dll"
-Repository: https://github.com/cake-contrib/Cake.CakeMail/
+Repository: https://github.com/cake-contrib/Cake.CakeMail
```

Comment:
@Jericho wondering if this is diff "enough" to warrant an update?

Reply:
We just need to decide what we want in our YAML files and I can make sure to standardize URLs. What do you think our standard should be: with or without slash?